Lib/calendar.py

```diff
@@ -734,10 +734,15 @@ def main(args=None):
         choices=("text", "html"),
         help="output type (text or html)"
     )
+    parser.add_argument(
+        "-f", "--first-weekday",
+        type=int, default=0,
+        help="weekday (0 is Monday, 6 is Sunday) to start each week (default 0)"
+    )
     parser.add_argument(
         "year",
         nargs='?', type=int,
-        help="year number (1-9999)"
+        help="year number"
```

Comment:
Why is it removed?

Reply:
The limit was removed in 66c88ce30ca.

Reply:
Good point.

This PR will only be merged in the main branch. Do you mind to create a separate PR that fixes this help for backporting?

Reply:
I _could_ make a separate PR.  How is the process different from a normal PR that goes into main?  This is my first PR for python.

Reply:
The simplest way -- you only need to create a trivial PR for main. I'll merge it and create backports, then I'll merge main into this branch and merge it into main.